### ISSUE TITLE
Variable substitution from shell environment

### ISSUE THREAD

**Issue by tonysickpony** [aws/amazon-ecs-cli#28]
Docker-compose enabled variable substitution from shell environment now. 
```
web:
    image: "myimage: ${GIT_COMMIT_HASH}"
    ports: 
        - '80:80'
...
```
It will be awesome if ecs-cli support this too. Or maybe this involves the issue with docker version running on ecs instance?

**Comment by ChristerWendel** [comment]
+1

**Comment by rodlogic** [comment]
+1

**Comment by christianblunden** [comment]
Docker compose actually let's you do this already.. so I guess the real question is how is the AWS ECS cli going to match the release schedule of docker-compose?

**Comment by uttarasridhar** [comment]
This is addressed in #121 and released in [v0.4.0](https://github.com/aws/amazon-ecs-cli/releases/tag/v0.4.0).
